Sources/SectionScrubber.swift

```diff
@@ -151,32 +162,34 @@ public class SectionScrubber: UIView {
         return (containingViewContentSize.height * percentageInView) - containingViewFrame.minY
     }
 
-    func handleLongPress(gestureRecognizer: UITapGestureRecognizer) {
-        self.sectionLabelState = gestureRecognizer.state == .Ended ? .Hidden : .Visible
-    }
+    func handleScrub(gestureRecognizer: UIGestureRecognizer){
```

Comment:
A space between the `)` and the `{` would be nice, it looks so tight right now. 😁